### DISCUSSION TITLE
已解决, 谢谢

### DISCUSSION THREAD

**Discussion by JW2006778** [getlantern/lantern-forum-cn#600] [General]
耐心即将耗尽。开始发售后邮件还回，现在连回都懒得回了。我的邮箱 ysq.wiscreet@gmail.com
烦请尽快解决 @Mike777Khoo

**Comment by MaxMKMax** [comment]
@JW2006778 您好. 以后有什么事来GitHub找我. 因为我这里没有权限管理邮件. 请您提供我支付单据和账号邮箱. 一般我会直接受理GitHub. 这样快一点.

**Reply by JW2006778** [reply]
支付收据
<img width="635" alt="截屏2024-06-17 09 32 55" src="https://github.com/getlantern/lantern-forum-cn/assets/9940420/77e936c6-1665-4564-82a1-7f23b3d5ad5b">
账号邮箱：ysq.wiscreet@gmail.com
（我想问一下，因为你们的程序bug或系统问题，让大家都集中在这里暴露信息真的好吗？？？）

**Reply by JW2006778** [reply]
免费版流量马上耗光，只怕到时候连邮箱都无法访问。@Mike777Khoo 你说怎么办？

**Reply by JW2006778** [reply]
我等不起了，24小时之内必须解决！！否则退款！！

**Reply by JW2006778** [reply]
我刚刚提交了退款申请，抗议售后对待同样的问题解决起来厚此薄彼！！10年老用户正式申请退款。

**Reply by MaxMKMax** [reply]
@JW2006778 https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
您好. 为了保护您的隐私. 请您发支付单据和账号邮箱到这个链接. 谢谢.

**Reply by MaxMKMax** [reply]
@JW2006778 我们已经改善提交问题方式了. 请您用这个link提交吧.

**Reply by JW2006778** [reply]
我已经提交、提交了两次（第一次是索取激活码，第二次是退款申请）。（在仍然有免费流量的情况下，我进入这个页用了10分钟）

**Comment by kennylo47** [comment]
系咯 把付款信息和邮箱都这样公开 我自己都觉得不妥。问题是即使这样问题还没有及时解决。

**Reply by MaxMKMax** [reply]
@kennylo47 https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
您好. 为了保护您的隐私. 请您发支付单据和账号邮箱到这个链接. 谢谢.

**Comment by MaxMKMax** [comment]
@JW2006778 我帮您再度提交问题上去了. 给我们 1-2 天的时间. 谢谢啦!

**Reply by JW2006778** [reply]
谢谢Mike。请退款吧，心塞。

**Reply by JW2006778** [reply]
两天的时间到了，退款进展如何？

**Reply by JOJO0915** [reply]
不知道他们周末是否也休息，我也是差不多时间提交的，目前没有任何回复

**Reply by MaxMKMax** [reply]
@JW2006778 您好. 客服 3 天前已经发邮件给您了. 请您检查垃圾邮箱或. 谢谢!

**Reply by JW2006778** [reply]
你好Mike。我检查了正常邮件文件夹、也检查了spam邮件夹，最近根本没有收到与蓝灯售后服务有关的任何邮件。
请你告诉我，这个Nolan发的邮件标题是什么、发送到了哪个邮件地址、他的始发邮件地址是哪个？

**Reply by JW2006778** [reply]
我需要提醒你们的是，我后边提出的售后请求都是要求退款的，并不像这个Nolan说的“它们是需要激活码”的，那是一周多以前的事了。请知悉。

**Reply by JW2006778** [reply]
Mike，你好。我刚刚又提交了一次退款申请，我明确告知你们，我不打算再使用蓝灯。请退款！

**Reply by MaxMKMax** [reply]
@JW2006778 嗯嗯, 那好吧. 很抱歉影响了您的体验感. 请您以后有机会再回来用
蓝灯. 您申请退款需要 14-20 天. 才能完成. 您提交之后, 客服那里会帮您解决了. 请放心.

**Reply by JW2006778** [reply]
Mike 你好！我在邮箱中通过过滤找到了Nolan发的激活码，激活了蓝灯，速度还行。退款请求正式撤销。
感谢你们做出的努力。另外，希望在iphone客户端方面尽快升级，谢谢！

**Reply by MaxMKMax** [reply]
@JW2006778 您好. 不客气呢. 可以帮助到您, 我非常开心. 谢谢您对我们的人工服务感到满意. 能服务到您是我们的荣幸! 很荣幸有您这么好的尊贵用户. 谢谢您决定撤销退款了. 希望蓝灯可以提供您更好的服务. 我们会尽全力优化软件来让您拥有最好的体验感. 非常感谢您支持蓝灯. 万分感谢!!!
